scripts/alert_emails/template.html

```diff
@@ -48,11 +48,13 @@
                                 style="font-size:26px;line-height:32px;Margin-top:30px;Margin-bottom:40px;color:#c3ced9;Margin-left:20px;Margin-right:20px;border-width:0!important;border-style:none!important;border-color:white!important">
                                 <div align="center" id="m_7975860115323008578emb-email-heaÞr"
                                     style="line-height:inherit;border-width:0!important;border-style:none!important;border-color:white!important">
-                                    <a href="https://covidactnow.org?utm_source=risk_alerts&utm_medium=email" style="line-height:inherit" target="_blank">
+                                    <a href="https://covidactnow.org?utm_source=risk_alerts&utm_medium=email"
+                                        style="line-height:inherit" target="_blank">
```

Comment:
Changes to this file are much smaller than what is displayed here. This is just the linter re-organizing white space!

Reply:
is there a setting in your linter that doesnt match CAN's?

Reply:
It might be! This is the second time it's happening so I'll take a look 👀